app/assets/stylesheets/leaderboard.css

```diff
@@ -250,6 +250,7 @@
   height: calc(6 * var(--fpx));
   border-radius: calc(3 * var(--fpx));
   transform-origin: left center;
+  transform: scaleX(var(--flight-bar-scale));
```

Comment:
I don't know where to put this comment... but the ends of the bars are no longer rounded - is that to be added in future?